artemis/reporting/modules/mail_dns_scanner/reporter.py

```diff
@@ -74,6 +76,22 @@ def create_reports(task_result: Dict[str, Any], language: Language) -> List[Repo
             )
         return result
 
+    @staticmethod
+    def get_alerts(all_reports: List[Report], reports_for_top_level_target_threshold: int = 8) -> List[str]:
+        result = []
+
+        reports_by_top_level_target_counter: CounterType[str] = Counter()
+        for report in all_reports:
+            if report.report_type == MailDNSScannerReporter.MISCONFIGURED_EMAIL:
+                reports_by_top_level_target_counter[report.top_level_target] += 1
```

Comment:
alternatively:

```suggestion
        targets = [r.top_level_target for r in all_reports if r.report_type == MailDNSScannerReporter.MISCONFIGURED_EMAIL]
        reports_by_top_level_target_counter = Counter(targets)
```
(but the current version is fine too, just quite imperative)